src/pagination/pagination.styles.tsx

```diff
@@ -12,13 +13,19 @@ import { TextStyleHelper } from "../text";
 interface StyleProps {
     $selected: boolean;
 }
-
+interface ButtonProps {
+    $position: string;
+}
 // =============================================================================
 // STYLING
 // =============================================================================
 
 export const PaginationWrapper = styled.div`
     display: flex;
+
+    ${MediaQuery.MaxWidth.mobileS} {
```

Comment:
not sure why this is needed?

Reply:
for iphone 5 like xsmall devices with first and last button its touching the left side